### PR TITLE
*: update docs about reset task and relay log purge

### PR DESCRIPTION
**Pull request by toutdesuite**
### What is changed, added or deleted? (Required)

- update the FAQ about reset task
- clarify the purge-expires config item

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v2.0 changes for now)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-dm/pull/103
- Other reference link(s):

### CODE REVIEW

File: en/error-handling.md [hunk 1]
```diff
@@ -85,7 +85,7 @@ If you encounter an error while running DM, take the following steps to troubles
     resume-task ${task name}
     ```
 
-However, you need to reset the data replication task in some cases. For details, refer to [Reset the Data Replication Task](faq.md#how-to-reset-the-data-replication-task).
+However, you need to reset the data replication task in some cases. For details, refer to [Reset the Data Replication Task](faq.md#how-to-reset-the-data-migration-task).
```
Comment (ran-huang) [inline en/error-handling.md:88]: @toutdesuite The linked title is not changed?

File: en/faq.md [hunk 2]
```diff
@@ -28,25 +28,18 @@ When you encounter a DDL statement unsupported by TiDB, you need to manually han
...
+3. Use one of the following ways to restart the data migration task.
 
-    - If the cluster is deployed using DM-Ansible, the relay log is in the `<deploy_dir>/relay_log` directory.
-    - If the cluster is manually deployed using the binary, the relay log is in the directory set in the `relay-dir` parameter.
-
-4. Clean up downstream replicated data.
-5. Use DM-Ansible to [start the entire DM cluster](deploy-a-dm-cluster-using-ansible.md#step-9-deploy-the-dm-cluster).
-6. Restart data replication with the new task name, or set `remove-meta` to `true` and `task-mode` to `all`.
+- Specify a new task name in the task configuration file. Then execute `start-task {task-config-file}`.
+- Execute `start-task --remove-meta {task-config-file}`.
```
Comment (ran-huang) [inline en/faq.md:42]: Please indent 4 spaces.

File: en/error-handling.md (hunk 1 above)
Comment (toutdesuite) [reply en/error-handling.md:88]: This doc has 24 `replication`, while `en/faq.md` 10 `migration`. I prefer using one expression to avoid ambiguity

Comment (ran-huang) [reply en/error-handling.md:88]: OK. I remember you saying something about unifying the usage of `replication` and `migration`. If there's any update, please let me know so we can unify them across all related docs in this repo.

Comment (toutdesuite) [reply en/error-handling.md]: @ran-huang Let's go with `replication` in `en/faq.md`. Regarding the usage of `replication` and `migration`, pls follow up on my proposal (shared the link in `Joyce 工作交接`. ) This PR will **not** unify the usage of `replication` and `migration`

**Comment by yikeke** [comment]
PTAL again @ran-huang

**Comment by ran-huang** [comment]
LGTM

**Comment by ran-huang** [comment]
@csuzhangxc PTAL

File: en/faq.md [hunk 3]
```diff
@@ -30,23 +30,16 @@ When you encounter a DDL statement unsupported by TiDB, you need to manually han
...
 
-You need to reset the entire data replication task in the following cases:
+When an exception occurs during data replication and the data replication task cannot be resumed, you need to reset the task and re-migrate the data:
 
-- `RESET MASTER` is accidentally executed in the upstream database.
-- The relay log or the upstream binlog is corrupted or lost.
+1. Execute the `stop-task` command to stop the abnormal data replication task.
 
-Generally, at this time, the relay unit exits with an error and cannot be automatically restored gracefully. You need to manually restore the data replication and the steps are as follows:
+2. Purge the data replicated to the downstream.
```
Comment (csuzhangxc) [inline en/faq.md:37]:
```suggestion
2. Clear the data replicated to the downstream.
```

Is `Clear` better here?

File: en/relay-log.md [hunk 4]
```diff
@@ -94,7 +94,7 @@ Method 1: Use the command-line options.
     - "3600" by default, indicating a background purge task is performed every 3600 seconds.
 
 + `purge-expires`
-    - The number of hours that a non-updated relay log can be retained for before being purged in the automatic background purge.
+    - The number of hours for which the relay log (that has not been written to the relay processing unit, and that is not being used or will not be read later by the currently running data migration task) can be retained before being purged in the automatic background purge.
```
Comment (csuzhangxc) [inline en/relay-log.md:97]: `当前 ** 没有写入` means `has been written previously` not `has not been written`.

File: en/faq.md (hunk 3 above)
Comment (ran-huang) [reply en/faq.md:37]: I think `purge` is better. We also use **data purge** in `relay-log.md`.

File: en/relay-log.md (hunk 4 above)
Comment (ran-huang) [reply en/relay-log.md:97]: The Chinese description is not so clear that it causes incorrect translation. Can you reword that sentence please?

Comment (ran-huang) [inline en/relay-log.md:97]:
```suggestion
    - The number of hours for which the relay log (that has been previously written to the relay processing unit, and that is not being used or will not be read later by the currently running data migration task) can be retained before being purged in the automatic background purge.
```
@csuzhangxc PTAL

File: en/faq.md (hunk 3 above)
Comment (csuzhangxc) [reply en/faq.md:37]: OK, but we use `purge` in `relay-log.md` because MySQL use it.

File: en/relay-log.md (hunk 4 above)
Comment (csuzhangxc) [reply en/relay-log.md:97]: LGTM

**Pull request merged**